main/ui/src/main/resources/fxml/vault_detail_locked.fxml

```diff
@@ -25,18 +24,20 @@
 				<FontAwesome5IconView glyph="KEY" glyphSize="15"/>
 			</graphic>
 		</Button>
-		<Hyperlink text="%main.vaultDetail.optionsBtn" onAction="#showVaultOptions">
+		<Label styleClass="label-small,label-muted" text="%main.vaultDetail.passwordSavedInKeychain" visible="${controller.passwordSaved}">
```

Comment:
Why not keep a hyperlink that opens the corresponding tab in vault options?

Reply:
> These would require bigger changes, should this be part of this PR?

Oh.

Yes I think so. Deep linking into a dialog can be copy-pasted from the app preferences, e.g. when an update is available.